src/controls/estimator_manager.hpp

```diff
@@ -19,7 +19,6 @@
 
 #define NUM_IMU_CALIBRATION 50000
 
-
 // Rev encoder pins

```

Comment:
Should this be moved into the rev encoder's driver files?

Reply:
nah, I don't think so, since we init the encoders in estimator manager.